src/components/webview/webviewEvents.ts

```diff
@@ -199,6 +199,10 @@ export type WebviewResponse =
 			oldFileContent: string;
 			newFileContent: string;
 	  }>
+	| Readonly<{
+			kind: 'webview.campaignManager.selectCase';
+			hash: string | null;
```

Comment:
can you unselect a case? if so, the `hash` should be just `CaseHash`

Reply:
Currently, the only way to unselect a case is to select another case. I think it's not worth supporting unselecting a case when there is only one case in the panel.

Reply:
Regardless, I refactored so that we have `hash: CaseHash;`